swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2019-12-31-preview/examples/OpenShiftClusters_CreateOrUpdate.json

```diff
@@ -12,27 +12,27 @@
       "properties": {
         "clusterProfile": {
           "domain": "cluster.location.aroapp.io",
-          "resourceGroupId": "/subscriptions/subscriptionId/resourceGroups/clusterResourceGroup"
+          "resourceGroupId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-cluster"
```

Comment:
I don't love the bleed between values used for test and the examples.  It's not the end of the world, but I don't love it.